### ISSUE TITLE
iPad usage?

### ISSUE THREAD

**Issue by marianmi** [iam4x/pokemongo-webspoof#425]
Is theree any way to bring the app on iPad, to use the iPad as a webspoofer? ;]

**Comment by kababoom** [comment]
Working fine if you got a not too old iPad...

**Comment by marianmi** [comment]
@kababoom How do you do this? 
Ok, it's no problem to bring the app on the iPad, but do you connect the phone with the iPad then, how??

**Comment by kababoom** [comment]
No no no, the Pokemon App runs fine on the iPad, the webspoofer of course needs to run on a mac with Xcode and the iPad connected via USB ...

**Comment by marianmi** [comment]
@kababoom yeah, but that's not what i wanted, i wanted to play outside, in the train, with two devices.
That's why I wrote 'iPad as a webspoofer'!  ;]

**Comment by maxfremars** [comment]
Its not possible to use the iPad instead of the computer. If, one time, the iPad Pro is able to run a dedicated Xcode App, it might be possible, but right now, you need a Mac with Xcode.
